src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/casetransfer/CaseTransferSameCountryService.java

```diff
@@ -26,6 +26,12 @@ public List<String> transferCase(CaseDetails caseDetails, String userToken) {
             throw new IllegalStateException("No cases found for Case Transfer for "
                     + caseDetails.getCaseData().getEthosCaseReference());
         }
+
+        // If transferring to the same office, skip validation and transfer directly
+        if (caseDetails.getCaseData().getOfficeCT().getSelectedCode()
```

Comment:
Extract caseData into its own variable here to make these calls a bit shorter? We're calling `getCaseData()` twice